.github/workflows/west-commands.yml

```diff
@@ -57,19 +57,77 @@ jobs:
         with:
           git-fetch-depth: 0
           path: nrf
-      - name: Install requirements
+      - name: Install requirements(bash)
         shell: bash
         run: |
+          pip3 install -r nrf/scripts/requirements-fixed.txt
           pip3 install -r nrf/scripts/requirements-west-ncs-sbom.txt
           pip3 install -r nrf/scripts/requirements-extra.txt
+      - name: Install requirements (Windows)
+        if: ${{ matrix.os == 'windows-latest' }}
+        shell: pwsh
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r nrf/scripts/requirements-fixed.txt
+          python -m pip install -r nrf/scripts/requirements-west-ncs-sbom.txt
+          python -m pip install -r nrf/scripts/requirements-extra.txt
```

Comment:
I thought requirements-extra.txt was included when populated requirements-fixed.txt, so is this line still needed ?